homeassistant/components/gardena_bluetooth/number.py

```diff
@@ -0,0 +1,146 @@
+"""Support for number entities."""
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+
+from gardena_bluetooth.const import DeviceConfiguration, Valve
+from gardena_bluetooth.parse import (
+    CharacteristicInt,
+    CharacteristicLong,
+    CharacteristicUInt16,
+)
+
+from homeassistant.components.number import (
+    NumberEntity,
+    NumberEntityDescription,
+    NumberMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import Coordinator, GardenaBluetoothEntity
+
+
+@dataclass
+class GardenaBluetoothNumberEntityDescription(NumberEntityDescription):
+    """Description of entity."""
+
+    char: CharacteristicInt | CharacteristicUInt16 | CharacteristicLong = field(
+        default_factory=lambda: CharacteristicInt("")
+    )
+
+
+DESCRIPTIONS = (
+    GardenaBluetoothNumberEntityDescription(
+        key=Valve.manual_watering_time.uuid,
+        translation_key="manual_watering_time",
+        native_unit_of_measurement="s",
```

Comment:
Please use our common unit time enum for the units.

Reply:
Will fix in followup. Not sure how i missed that.

Reply:
Fixed in https://github.com/home-assistant/core/pull/96683